docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -622,7 +622,7 @@ part of the server.
 
 When using Vault as a backend you can share configuration with
 all applications by placing configuration in
-`{backend}/application`.  For example, if you run this Vault command
+`secret/application`.  For example, if you run this Vault command
```

Comment:
Thats a good point, and we also specify that by default the backend is `secret` and how to change it, so I guess this makes sense.  Thanks!